README.md

```diff
@@ -166,9 +166,24 @@ spec:
 | Private Builder Image Registry     | ⚪️     |      |    |
 | Runtime Base Image    | ⚪️    |      |    |
 | Binary builds         |       |      |    |
+| Image pull policy     |       |      |    |
+| Inject source via config map         |       |      |    |
+| Inject source via secret         |       |      |    |
+| Inject source via image         |       |      |    |
+| Set image label/ens   |       |      |    |
```

Comment:
I would envision this being something buildv2 does solely by leveraging tekton

Reply:
this is setting labels on the output?  what's ens?  environment variables?  if so env variables would be inputs right?

Reply:
yeah this needs to change to `Set specified labels on output image` ... the `ens` was a leftover I thought I had deleted prior to branch push / PR creation